packages/vision/src/helpers.js

```diff
@@ -196,6 +210,9 @@ module.exports = apiVersion => {
    *
    *   If the key is `content`, the value should be a Buffer.
    * @param {Object=} options
+   *   Any additional options which should be mapped onto the
```

Comment:
Concur on fixing the documentation. I will do that after we resolve your other comment.